4.0.10 FTBFS

```
[   14s] Executing(%build): /bin/sh -e /var/tmp/rpm-tmp.GDUj4T
[   14s] + umask 022
[   14s] + cd /home/abuild/rpmbuild/BUILD
[   14s] + /usr/bin/rm -rf /home/abuild/rpmbuild/BUILDROOT/muparserx-4.0.10-0.x86_64
[   14s] ++ dirname /home/abuild/rpmbuild/BUILDROOT/muparserx-4.0.10-0.x86_64
[   14s] + /usr/bin/mkdir -p /home/abuild/rpmbuild/BUILDROOT
[   14s] + /usr/bin/mkdir /home/abuild/rpmbuild/BUILDROOT/muparserx-4.0.10-0.x86_64
[   14s] + cd muparserx-4.0.10
[   14s] + find . -name CMakeLists.txt -exec sed -i -re '/^[[:blank:]]*[sS][eE][tT][[:blank:]]*\([[:blank:]]*(CMAKE_BUILD_TYPE|CMAKE_COLOR_MAKEFILE|CMAKE_INSTALL_PREFIX|CMAKE_VERBOSE_MAKEFILE).*\)/{s/^/#IGNORE /}' '{}' +
[   14s] + mkdir -p build
[   14s] + cd build
[   14s] + /usr/bin/cmake /home/abuild/rpmbuild/BUILD/muparserx-4.0.10/. '-GUnix Makefiles' -DCMAKE_INSTALL_PREFIX:PATH=/usr -DINCLUDE_INSTALL_DIR:PATH=/usr/include -DLIB_INSTALL_DIR:PATH=/usr/lib64 -DSYSCONF_INSTALL_DIR:PATH=/etc -DSHARE_INSTALL_PREFIX:PATH=/usr/share -DCMAKE_INSTALL_LIBDIR:PATH=lib64 -DCMAKE_INSTALL_LIBEXECDIR=/usr/libexec -DCMAKE_BUILD_TYPE=RelWithDebInfo '-DCMAKE_C_FLAGS=-O2 -Wall -D_FORTIFY_SOURCE=2 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -Werror=return-type -flto=auto -g -DNDEBUG' '-DCMAKE_CXX_FLAGS=-O2 -Wall -D_FORTIFY_SOURCE=2 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -Werror=return-type -flto=auto -g -DNDEBUG' '-DCMAKE_Fortran_FLAGS=-O2 -Wall -D_FORTIFY_SOURCE=2 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -Werror=return-type -flto=auto -g -DNDEBUG' '-DCMAKE_EXE_LINKER_FLAGS=-flto=auto -Wl,--as-needed -Wl,--no-undefined -Wl,-z,now' '-DCMAKE_MODULE_LINKER_FLAGS=-flto=auto -Wl,--as-needed' '-DCMAKE_SHARED_LINKER_FLAGS=-flto=auto -Wl,--as-needed -Wl,--no-undefined -Wl,-z,now' -DLIB_SUFFIX=64 -DCMAKE_VERBOSE_MAKEFILE:BOOL=ON -DBUILD_SHARED_LIBS:BOOL=ON -DBUILD_STATIC_LIBS:BOOL=OFF -DCMAKE_COLOR_MAKEFILE:BOOL=OFF -DCMAKE_INSTALL_DO_STRIP:BOOL=OFF -DCMAKE_MODULES_INSTALL_DIR=/usr/lib64/cmake/muparserx -DBUILD_SHARED_LIBS:BOOL=ON -DBUILD_SAMPLES:BOOL=OFF -DCMAKE_BUILD_TYPE:STRING=Release
[   14s] -- The CXX compiler identification is GNU 11.2.1
...
[   15s] -- Build files have been written to: /home/abuild/rpmbuild/BUILD/muparserx-4.0.10/build
[   15s] + make VERBOSE=1 -j16
[   15s] /usr/bin/cmake -S/home/abuild/rpmbuild/BUILD/muparserx-4.0.10 -B/home/abuild/rpmbuild/BUILD/muparserx-4.0.10/build --check-build-system CMakeFiles/Makefile.cmake 0
[   15s] /usr/bin/cmake -E cmake_progress_start /home/abuild/rpmbuild/BUILD/muparserx-4.0.10/build/CMakeFiles /home/abuild/rpmbuild/BUILD/muparserx-4.0.10/build//CMakeFiles/progress.marks
...
[   16s] [ 67%] Building CXX object CMakeFiles/muparserx.dir/parser/mpParser.cpp.o
[   16s] /usr/bin/c++ -Dmuparserx_EXPORTS -I/home/abuild/rpmbuild/BUILD/muparserx-4.0.10/parser -O2 -Wall -D_FORTIFY_SOURCE=2 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -Werror=return-type -flto=auto -g -DNDEBUG -pedantic -Wall -Wextra -std=c++11 -O3 -DNDEBUG -fPIC -MD -MT CMakeFiles/muparserx.dir/parser/mpParser.cpp.o -MF CMakeFiles/muparserx.dir/parser/mpParser.cpp.o.d -o CMakeFiles/muparserx.dir/parser/mpParser.cpp.o -c /home/abuild/rpmbuild/BUILD/muparserx-4.0.10/parser/mpParser.cpp
[   16s] /home/abuild/rpmbuild/BUILD/muparserx-4.0.10/parser/mpPackageCommon.cpp:37:10: fatal error: mpOprtBinShortCut.h: No such file or directory
[   16s]    37 | #include "mpOprtBinShortCut.h"
[   16s]       |          ^~~~~~~~~~~~~~~~~~~~~
[   16s] compilation terminated.
[   16s] make[2]: *** [CMakeFiles/muparserx.dir/build.make:401: CMakeFiles/muparserx.dir/parser/mpPackageCommon.cpp.o] Error 1
```

--- a/parser/mpPackageCommon.cpp
+++ b/parser/mpPackageCommon.cpp
@@ -34,7 +34,7 @@
 #include "mpFuncNonCmplx.h"
 #include "mpFuncCommon.h"
 #include "mpOprtBinCommon.h"
-#include "mpOprtBinShortCut.h"
+#include "mpOprtBinShortcut.h"
 #include "mpOprtBinAssign.h"
 #include "mpOprtPostfixCommon.h"
 #include "mpValReader.h"

--- a/parser/mpParserBase.cpp
+++ b/parser/mpParserBase.cpp
@@ -43,7 +43,7 @@
 #include "mpDefines.h"
 #include "mpIfThenElse.h"
 #include "mpScriptTokens.h"
-#include "mpOprtBinShortCut.h"
+#include "mpOprtBinShortcut.h"
 
 using namespace std;
 

